### DISCUSSION TITLE
Gradio username and password

### DISCUSSION THREAD

**Discussion by faffrd** [oobabooga/text-generation-webui#158] [Q&A] [answered]
Is there a way to set a gradio username and password like in automatic1111?
I tried --gradio-auth User/PW and that didn't work.

**Comment by noarche** [comment, marked as answer]
You need to put a path to a text file, inside the textfile store USER:PASS

One account per line allowing you to have multiple accounts. 

It should look similar to this:
**--listen --gradio-auth C:\New-File-With_userpass.txt**
